sharding/nodesCoordinator/indexHashedNodesCoordinatorLite_test.go

```diff
@@ -0,0 +1,89 @@
+package nodesCoordinator
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfo(t *testing.T) {
+	t.Parallel()
+
+	arguments := createArguments()
+
+	ihgs, err := NewIndexHashedNodesCoordinator(arguments)
+	require.Nil(t, err)
+	epoch := uint32(1)
+
+	randomness := []byte("rand seed")
+
+	ihgs.nodesConfig[epoch] = ihgs.nodesConfig[0]
+
+	body := createBlockBodyFromNodesCoordinator(ihgs, epoch)
+	validatorsInfo, err := createValidatorInfoFromBody(body, arguments.Marshalizer, 10)
+	require.Nil(t, err)
+
+	err = ihgs.SetNodesConfigFromValidatorsInfo(epoch, randomness, validatorsInfo)
+
+	validators, err := ihgs.GetAllEligibleValidatorsPublicKeys(epoch)
+	require.Nil(t, err)
+	require.NotNil(t, validators)
+}
+
+func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfoMultipleEpochs(t *testing.T) {
+	t.Parallel()
+
+	arguments := createArguments()
+
+	ihgs, err := NewIndexHashedNodesCoordinator(arguments)
+	require.Nil(t, err)
+	epoch := uint32(1)
+
+	randomness := []byte("rand seed")
+
+	ihgs.nodesConfig[epoch] = ihgs.nodesConfig[0]
+
+	body := createBlockBodyFromNodesCoordinator(ihgs, epoch)
+	validatorsInfo, err := createValidatorInfoFromBody(body, arguments.Marshalizer, 10)
+	require.Nil(t, err)
+
+	ihgs.SetNodesConfigFromValidatorsInfo(epoch, randomness, validatorsInfo)
```

Comment:
"_Unhandled error_" or each `SetNodesConfigFromValidatorsInfo` func call. Perhaps refactor it to `err = ihgs.SetNodesConfigFromValidatorsInfo(...)` and check for error to be nil?

Reply:
yea. initially, the function returned nothing, so I missed to change later on in all appearances, added the nil check :+1: